api.planx.uk/modules/send/routes.ts

```diff
@@ -17,11 +18,37 @@ router.post(
   validate(combinedEventsPayloadSchema),
   createSendEvents,
 );
-router.post("/bops/:localAuthority", useHasuraAuth, sendToBOPS);
-router.post("/uniform/:localAuthority", useHasuraAuth, sendToUniform);
-router.post("/idox/:localAuthority", useHasuraAuth, sendToIdoxNexus);
-router.post("/email-submission/:localAuthority", useHasuraAuth, sendToEmail);
+router.post(
```

Comment:
If these endpoints were grouped at `/send` or similar, we could just apply the `useHasuraAuth` and `validate()` middleware once at this level with a `router.use()`.

This change would involve an update to the Hasura events also, so this can be a discrete PR in future.